core/storage/classifier/gae_models.py

```diff
@@ -180,18 +180,18 @@ def query_new_and_pending_training_jobs(cls, offset=0):
 
         Returns:
             tuple(list(ClassifierTrainingJobModel), int).
-            list(ClassifierTrainingJobModel). List of the
-            ClassifierTrainingJobModels with status new or pending.
-            int. The offset value.
+            List of the ClassifierTrainingJobModels with status new or pending.
+            The offset value.
         """
+        fetch_limit = 10
```

Comment:
No, put this at a top of this file as a constant, the name should also convey for what this is the fetch limit.

Reply:
Done